docs/upgrade-to-2.0.md

```diff
@@ -0,0 +1,42 @@
+# Upgrading to FirebaseUI 2.0
+
+This document outlines the _breaking_ changes made between versions 1.2.0 and 2.0.0 as well as
+suggested upgrade paths. This is not a comprehensive list of all features or bugfixes, please
+reference the release note for that information.
+
+For a full, machine-generated compatibility report see the `docs/compat_reports` folder.
+
+## Auth
+
+  * **Facebook and Twitter Dependencies** - FirebaseUI no longer directly depends on the Facebook
+    or Twitter SDKs. If you plan to enable either of these providers you should include the
+    appropriate SDK in your app to avoid a crash. This saves space for developers only using
+    Google/Email/Phone authentication.
+  * **Sign Out and Delete** - The `AuthUI#delete(Activity)` and `AuthUI#signOut(Activity)` 
+    methods have been removed, please use the versions that accept `FragmentActivity`.
+  * **Error Codes** - The `ResultCodes` class has been removed, instead in `onActivityResult` you 
```

Comment:
Oh interesting.  We did do something about the network error code though, do you remember what that change was?